src/pal/src/exception/seh-unwind.cpp

```diff
@@ -491,7 +491,17 @@ BOOL PAL_VirtualUnwindOutOfProc(CONTEXT *context,
 
     LibunwindCallbacksInfo.Context = context;
     LibunwindCallbacksInfo.readMemCallback = readMemCallback;
+#if UNWIND_CONTEXT_IS_UCONTEXT_T
     WinContextToUnwindContext(context, &unwContext);
+#else
+    st = unw_getcontext(&unwContext);
```

Comment:
@janvorli In the case of a dummy unwContext, it theoretically would be possible for the VA of the stack to collide and cause us to exclude a valid context pointer -- no?